go.mod

```
@@ -78,7 +78,7 @@ require (
 	github.com/google/go-containerregistry v0.20.3
 	github.com/iamacarpet/go-win64api v0.0.0-20240507095429-873e84e85847
 	github.com/k3s-io/helm-controller v0.16.13
-	github.com/k3s-io/k3s v1.33.3-0.20250715012419-cf2ef56313e8 // master
+	github.com/k3s-io/k3s v1.33.4-0.20250808100931-6382d6a5fc23 // master
```

Nit: no longer master 
```suggestion
	github.com/k3s-io/k3s v1.33.4-0.20250808100931-6382d6a5fc23 // release-1.33
```